docs/Getting-Started-frontend.md

```diff
@@ -303,6 +304,12 @@ meta: {
 
 ----
 
+#### How does the Logical layer (Model) of the website works?
+
+Read [Information-Flow.md](Information-Flow.md).
```

Comment:
Since this "Getting Started" guide isn't specific to Group Income, but a generic guide for using frameworks like React and Vue, I recommend deleting this section and instead linking to the Information Flow document either from the README (as a bullet underneath the Style Guide), or from the Style Guide itself.

Reply:
Hum... I disagree. There's a bunch of other common questions related to the architecture of GI so this is just one more question. However, I'll add it to README as well.